pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
> And it requires that something to bypass any authorization checks so you don't end up in infinite loops, so you can't pass it a normal client.

Oh, and either way you do it, you'll need a path that bypasses the escalation check to be able to prime your etcd with initial data that is by nature escalating.  In openshift, we have a Registry sort of interface with an `CreateRoleWithEscalation` kind of method to make that work (https://github.com/openshift/origin/blob/master/pkg/authorization/registry/role/policybased/virtual_storage.go#L108).